### ISSUE TITLE
I cant seem to figure out the webhook/making it for discord can someone help

### ISSUE THREAD

**Issue by Jano1g** [amprocode/Bloxflip-rain-notifier#29]

**Comment by luxkatana** [comment]
:wave: Hey there!

You'll need a **discord server** and a **text-channel** with in the discord server.

Go to **Channel settings(Edit Channel when hovering on the channel)** > **Integrations** > There should be an option to **Create a new webhook.**
![image](https://github.com/amprocode/Bloxflip-rain-notifier/assets/57036931/d74f004d-d840-4901-9ee6-0b72336668cb)

Create one, it will create a new webhook with a stupid name
![image](https://github.com/amprocode/Bloxflip-rain-notifier/assets/57036931/af11d3e0-9355-4e58-8e15-06264f83493a)

click the **Copy Webhook URL** and you are ready! 

PS: don't giveyour webhook URL to someone else, you'll regret it. Trust me**

**Comment by Jano1g** [comment]
it is safe tho to use the webhook with this right?

**Comment by amprocode** [comment]
Yeah, code is open source so you can have a look and see what it does

**Comment by luxkatana** [comment]
> it is safe tho to use the webhook with this right?

It is safe to run.

**Comment by amprocode** [comment]
Yes

**Comment by luxkatana** [comment]
If this issue has been solved, please close this issue.
